### PR TITLE
Wholistic Document Review PR

### PR DESCRIPTION
**Pull request by VoR0220**
This so far contains tidbits I garnered while working my way through the docs. Awaiting discussions on whether to pursue something a bit more substantial that could manifest in the form of a quick start or a more hand holdy guide for users. 

Update: It's been decided to do the quick start in a separate PR. This will suffice for now.

### CODE REVIEW

**Review by excalq — APPROVED** [review]

File: docs/userGuides/tags/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Tag Information Documentation: 
 
-The protocol uses tags to assess fees and facilitate rule checks for addresses. Tags are applied to addresses via an addTag function in the app manager by application administators. Tags are stored as bytes32 in a mapping inside the tags data contract. A maximum of 10 tags may be applied to each address. 
+The protocol uses tags to assess fees and facilitate rule checks for addresses. Tags are applied to addresses via an addTag function in the app manager by application administators. Tags are stored as bytes32 in a mapping inside the tags data contract. A maximum of 10 tags may be applied to each address. Their purpose is to be for application of rulesets to specific groups of individuals that may be too narrow to be encompassed in an access level.
```
Comment (jjbailey-io) [inline docs/userGuides/tags/README.md:7]: Maybe reword to:
`Tags allow application administrators to divide users into segments so a particular rule only applies to specific segments of users. The applicability will vary depending on the rule so see the documentation for each rule to understand how and when tags will apply.`
